scripts/generate/consts.go

```diff
@@ -79,6 +85,32 @@ func generateStickerTypeConsts(d APIDescription) (string, error) {
 	return out.String(), nil
 }
 
+func generateChatTypeConsts(d APIDescription) (string, error) {
```

Comment:
Could/should it be factorised together with with `generateStickerTypeConsts()`, which looks like almost identical to me ?

Reply:
I think it's better we keep it as simple as possible and write separated functions for them.
Also, other constant types do have their own functions for auto-generating, so I did what was already being done in the codebase.

Reply:
Let's refactor it later, and see if this works first. 

Do you want to regenerate the library so we can see the changes actually appear in the code? Docs are in the README!

Reply:
Right, regenerated the code as well